apps/shared-services/seo.service.ts

```diff
@@ -73,9 +73,10 @@ export class SeoService {
   }
 
   noIndex(value: boolean) {
+    const host = window.location.hostname;
```

Comment:
This is incorrect, no index needs to be called in the constructor itself, just like it is done for bot=true